google/cloud/apigeeregistry/v1/consistency_report.proto

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
Nit: 2022

I'd prefer to not put these directly in the "v1" directory - that would suggest that they are an official part of the API. Application-specific protos are in subdirectories (`apihub`, `style`, etc). I think this could go into `style` but that it's different enough to have its own directory that we might call `consistency`.